src/github_client.py

```diff
@@ -280,8 +280,13 @@ def get_job_info(self, path: GithubRepo, workflow_run_id: str, runner_name: str)
                         # though we would assume that it should always be present,
                         # as the job should be finished
                         conclusion = job.get("conclusion", None)
+
+                        job_id = job["id"]
```

Comment:
I am currently refactoring the various API interfaces.
I think this would be better to be a part of that.
I added this comment link to our tracking system.